lib/check_category_mapping.json

```diff
@@ -0,0 +1,208 @@
+  "no-fallthrough": "Bug Risk",
+  "no-floating-decimal": "Clarity",
+  "no-func-assign": "Bug Risk",
+  "no-implicit-coercion": "Bug Risk",
+  "no-implied-eval": "Security",
+  "no-inner-declarations": "Compatibility",
+  "no-invalid-regexp": "Bug Risk",
+  "no-invalid-this": "Bug Risk",
+  "no-irregular-whitespace": "Compatibility",
+  "no-iterator": "Compatibility",
+  "no-label-var": "Bug Risk",
+  "no-labels": "Bug Risk",
+  "no-lone-blocks": "Bug Risk",
+  "no-loop-func": "Bug Risk",
+  "no-magic-numbers": "Clarity",
+  "no-mixed-requires": "Clarity",
+  "no-multi-spaces": "Bug Risk",
+  "no-multi-str": "Compatibility",
+  "no-native-reassign": "Bug Risk",
+  "no-negated-in-lhs": "Bug Risk",
+  "no-new": "Bug Risk",
+  "no-new-func": "Clarity",
+  "no-new-require": "Clarity",
+  "no-new-wrappers": "Bug Risk",
+  "no-obj-calls": "Bug Risk",
+  "no-octal": "Compatibility",
+  "no-octal-escape": "Compatibility",
+  "no-param-reassign": "Bug Risk",
+  "no-path-concat": "Bug Risk",
+  "no-process-env": "Bug Risk",
+  "no-process-exit": "Bug Risk",
+  "no-proto": "Compatibility",
+  "no-redeclare": "Bug Risk",
+  "no-regex-spaces": "Bug Risk",
+  "no-restricted-modules": "Security",
+  "no-return-assign": "Bug Risk",
+  "no-script-url": "Security",
+  "no-self-compare": "Bug Risk",
+  "no-sequences": "Bug Risk",
+  "no-shadow": "Bug Risk",
+  "no-shadow-restricted-names": "Bug Risk",
+  "no-sparse-arrays": "Bug Risk",
+  "no-sync": "Bug Risk",
+  "no-throw-literal": "Clarity",
+  "no-undef": "Bug Risk",
+  "no-undef-init": "Bug Risk",
+  "no-undefined": "Compatibility",
+  "no-unexpected-multiline": "Bug Risk",
+  "no-unreachable": "Bug Risk",
+  "no-unused-expressions": "Bug Risk",
+  "no-unused-vars": "Bug Risk",
+  "no-use-before-define": "Compatibility",
+  "no-useless-call": "Bug Risk",
+  "no-useless-concat": "Bug Risk",
+  "no-void": "Compatibility",
+  "no-warning-comments": "Bug Risk",
+  "no-with": "Compatibility",
+  "np-dupe-keys": "Bug Risk",
+  "radix": "Bug Risk",
+  "use-isnan": "Bug Risk",
+  "use-strict": "Bug Risk",
+  "valid-jsdoc": "Clarity",
+  "valid-typeof": "Bug Risk",
+  "vars-on-top": "Clarity",
+  "wrap-iife": "Clarity",
+  "yoda": "Clarity",
+
+  "jsdoc/check-param-names": "Clarity",
+  "jsdoc/check-tag-names": "Clarity",
+  "jsdoc/check-types": "Clarity",
+  "jsdoc/newline-after-description": "Clarity",
+  "jsdoc/require-description-complete-sentence": "Clarity",
+  "jsdoc/require-hyphen-before-param-description": "Clarity",
+  "jsdoc/require-param": "Clarity",
+  "jsdoc/require-param-description": "Clarity",
+  "jsdoc/require-param-type": "Clarity",
+  "jsdoc/require-returns-description": "Clarity",
+  "jsdoc/require-returns-type": "Clarity",
+
+  "lodash/callback-binding": "Compatibility",
+  "lodash/collection-method-value": "Bug Risk",
+  "lodash/collection-return": "Bug Risk",
+  "lodash/no-double-unwrap": "Bug Risk",
+  "lodash/no-extra-args": "Clarity",
+  "lodash/unwrap": "Bug Risk",
```

Comment:
This looks great. I like the plugin-specific sections. Thanks!

Would you mind running these sections through an alphabetical sort? Looks like they're a bit out of order.